src/CSharpMongoMigrations/Extensions/BsonDocumentExtensions.cs

```diff
@@ -54,7 +53,22 @@ public static void AddProperty(this BsonDocument document, string property, obje
         /// <remarks>Id is always a Guid value by convention.</remarks>
         public static void AddUniqueIdentifier(this BsonDocument document, Guid value)
         {
-            document.AddProperty("_id", value);
+            var id = new BsonBinaryData(value, GuidRepresentation.Standard);
```

Comment:
Hi @rrmanzano. Thank you for the PR.

I guess there is no need to repeat the code here. You can easily call the polymorphic method and pass `GuidRepresentation.Standard` as the second parameter.